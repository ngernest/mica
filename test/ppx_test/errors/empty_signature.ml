module type S = sig end [@@deriving mica_types, mica]
