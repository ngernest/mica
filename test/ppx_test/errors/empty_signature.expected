module ExprToImpl(M : S) = struct end 